City-of-Helsinki/drupal-helfi-platform-config
# UHF-2938: Add image paragraph to accordion item

How to test with platform:

1. `composer create-project City-of-Helsinki/drupal-helfi-platform:dev-main hel-platform --no-interaction --repository https://repository.drupal.hel.ninja/`
2. `cd hel-platform`
3. `composer require drupal/hdbt:dev-UHF-2938_image_paragraph_to_accordion && composer require drupal/helfi_platform_config:dev-UHF-2938_image_paragraph_to_accordion`
4. `make new`
5. Log in to the site using the link provided.
6. Go to add new content. For example basic page will do fine. For the page give it a title and add accordion for upper or lower content region.
7. For the accordion you should now have ability to add image paragraph in addition to text and columns.
8. Add image paragraph and some other content too and make sure the page and the paragraph look fine.

Also check these PRs:
https://github.com/City-of-Helsinki/drupal-hdbt/pull/186
https://github.com/City-of-Helsinki/drupal-helfi-kymp/pull/193
https://github.com/City-of-Helsinki/drupal-helfi-strategia/pull/158
https://github.com/City-of-Helsinki/drupal-helfi-sote/pull/221
